src/aditiGJUN20/AddPositiveNumbers.java

```diff
@@ -0,0 +1,28 @@
+package aditiGJUN20;
+
+import java.util.Scanner;
+
+public class AddPositiveNumbers {
+
+	public static void main(String[] args) {
+		Scanner scanner=new Scanner(System.in);
+		int sum=0;
+		int count =1;
+		
+		
```

Comment:
Please remove extra blank line from next time.

Reply:
Suggested changes are done and pushed the code.